examples/app.ts

```diff
@@ -4,9 +4,9 @@
 /* eslint-disable no-console */
 import fs from 'fs';
 import path from 'path';
-import { Buffer } from 'buffer';
 import { getLocalAlgodClient, getLocalAccounts, compileProgram } from './utils';
 import algosdk from '../src';
+import { base64ToBytes, base64ToString } from '../src/utils/utils';
```

Comment:
Since you are importing these in the examples, that shows these conversion functions are going to be very useful to consumers as well.

For that reason, I think it would be a good idea to export them to be easily used by consumers. We already have the `src/encoding` directory where we put encoding functions which are later exported (see `src/main.ts`), and I think these would fit in there really well.

Reply:
I'm on board with this - I separated it out into `binarydata.ts` under `encoding/` and exported it in `main.ts`